src/pages/[slug].tsx

```diff
@@ -1,17 +1,17 @@
-import 'regenerator-runtime/runtime';
+// import 'regenerator-runtime/runtime';
 import fs from 'fs';
 import path from 'path';
 import matter from 'gray-matter';
 import { serialize } from 'next-mdx-remote/serialize';
 import Prayer from 'components/layout/PrayerPage';
 import Layout from 'components/layout/Layout';
 import { PRAYERS_PATH, prayersFilePaths } from 'utils/mdxUtils';
-import { useWakeLock } from 'use-wake-lock';
+// import { useWakeLock } from 'use-wake-lock';
 
 export default function PrayerPage({ prayer }) {
-  const { toggleWakeLock } = useWakeLock();
+  // const { toggleWakeLock } = useWakeLock();
 
-  toggleWakeLock();
+  // toggleWakeLock();
```

Comment:
I'm not removing this completely, I'll get back to this later when PWA is merged